Notes.md

```diff
@@ -6,4 +6,14 @@
 
 ```bash
 echo "This is interesting.!"
-```
\ No newline at end of file
+```
+
+Q) What is Git?
+- Git is a Distributed Version Control tool that is used to store different versions of a file in a remote or local repository. It is used to track changes in the source code. It allows multiple developers to work together. A VCS allows you to keep every change you make in the code repository.
+
+Q) What is the difference between git & GitHub?
+- Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
+
+Q) What is meant by DevOps?
+- DevOps is a software development strategy which bridges the gap between the development team and the Ops team of the company. 
+- The goal of DevOps is to change and improve the relationship by advocating better communication and collaboration between these two business units.
```

Comment:
Hmm interesting point here!

The way I think is that `DevOps` is the branch of software engineering which facilitates application development by software engineers, the process of software distribution and scalability.

Reply:
Correct me if I am wrong, so you mean DevOps facilitates software engineers to manage all the phases of software development like Plan -> Code -> Build -> Deploy along with infrastructure management? Well, what I have understood so far is that DevOps is like a methodology that helps to bridge the gap between the Dev team and the Ops team. So they can work seamlessly and provide faster resolution of problems. As a result of this improved communication and collaboration business benefits in faster delivery and more stable operating environments.

Reply:
> DevOps facilitates software engineers to manage all the phases of software development like Plan -> Code -> Build -> Deploy along with infrastructure management? 

Yes, I do think that you've summarized it accurately. 

> DevOps is like a methodology that helps to bridge the gap between the Dev team and the Ops team

This is true, however, in a startup setting you'll also be working with founders/ product manager to ensure things like the cost of `networking` needs to lowered for the website visitors in India etc. Of course, your collaboration would be there with the `BackEnd` team as well but these days people are doing `serverless` infrastructure which means a FrontEnd engineer ( Web or Mobile )  and a DevOps person is really enough to launch, sustain and even scale a business.

Here are some links you might find useful
https://aws.amazon.com/lambda/
https://aws.amazon.com/amplify/

And Google has another similar product
https://firebase.google.com/


In my limited experience, I've observed that two software engineering roles are very much highlighted in a startup - FrontEnd engineering and DevOps engineering. The former takes care of `what the user will see - how the business logic works`, the latter takes care of how do we get this `user interface` to the user in a `cost effective` way.

In short, @sharma-shruti , you'd find it more empowering to adapt a start-up/end-user oriented mindset rather than the traditional corporate DevOps philosophy. Companies, especially the ones with startup mindset, want people who take ownership at product level, which means people who think about improving the end-user experience. This will help you connect more with Founders and Core team.

Reply:
@arushi-bhatt , would you like to add your thoughts here?